src/multibody/joint/joint-basic-visitors.hxx

```diff
@@ -232,6 +232,28 @@ namespace se3
     return JointNeutralConfigurationVisitor::run(jmodel);
   }
 
+  /**
+   * @brief      JointNormalizeVisitor visitor
+   */
+  class JointNormalizeVisitor: public boost::static_visitor<>
+  {
+  public:
+    Eigen::VectorXd & q;
+
+    JointNormalizeVisitor(Eigen::VectorXd & q) : q(q) {}
+
+    template<typename D>
+    void operator()(const JointModelBase<D> & jmodel) const
+    { jmodel.normalize(q); }
```

Comment:
Does any know why method `void operator()(const JointModelBase<D> & jmodel) const` compiles without error ?

The method is const, so variable `q` should be of type `const Eigen::VectorXd &` and JointModelBase<D>::normalize(const  Eigen::VectorXd &)` does not exists.
I must be missing something obvious...